java/java/test/run

```diff
@@ -0,0 +1,155 @@
+
+#!/bin/bash
+#
+# The 'run' performs a simple test that verifies the S2I image.
+# The main focus here is to exercise the S2I scripts.
+#
+# For more information see the documentation:
+# https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+IMAGE_NAME=${IMAGE_NAME-s2i-test-candidate}
```

Comment:
Yeah, that is weird.  Same in the py case.  Copied from the s2i sample project test.